Runtimes/Overlay/cmake/modules/InstallSwiftInterface.cmake

```
@@ -1,46 +1,13 @@
 
 # Install the generated swift interface files for the target.
 function(install_swift_interface target)
-  # Install binary swift modules
-  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule"
-    RENAME "${SwiftOverlay_MODULE_TRIPLE}.swiftmodule"
-    DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
-  if(SwiftOverlay_VARIANT_MODULE_TRIPLE)
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}-${SwiftOverlay_VARIANT_MODULE_TRIPLE}/${target}.swiftmodule"
-      RENAME "${SwiftOverlay_VARIANT_MODULE_TRIPLE}.swiftmodule"
-      DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
+  # Swiftmodules are already in the directory structure
+  get_target_property(module_name ${target} Swift_MODULE_NAME)
+  if(NOT module_name)
+    set(module_name ${target})
   endif()
 
-  # Install Swift interfaces if library-evolution is enabled
-  if(SwiftOverlay_ENABLE_LIBRARY_EVOLUTION)
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftinterface"
-      RENAME "${SwiftOverlay_MODULE_TRIPLE}.swiftinterface"
-      DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
-
-    install(FILES "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.private.swiftinterface"
-      RENAME "${SwiftOverlay_MODULE_TRIPLE}.private.swiftinterface"
-      DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
-
-    # Install catalyst interface files
-    if(SwiftOverlay_VARIANT_MODULE_TRIPLE)
-      install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}-${SwiftOverlay_VARIANT_MODULE_TRIPLE}/${target}.swiftinterface"
-        RENAME "${SwiftOverlay_VARIANT_MODULE_TRIPLE}.swiftinterface"
-        DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
-      install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${target}-${SwiftOverlay_VARIANT_MODULE_TRIPLE}/${target}.private.swiftinterface"
-        RENAME "${SwiftOverlay_VARIANT_MODULE_TRIPLE}.private.swiftinterface"
-        DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule")
-    endif()
-  endif()
-
-  # Install Swift documentation interface files.
-  install(FILES "${CMAKE_CURRENT_BINARY_DIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftdoc"
-    RENAME "${SwiftOverlay_MODULE_TRIPLE}.swiftdoc"
-    DESTINATION "${SwiftOverlay_INSTALL_SWIFTMODULEDIR}/$<TARGET_PROPERTY:${target},Swift_MODULE_NAME>.swiftmodule"
+  install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${module_name}.swif\tmodule"
```

Added a stray `\`, which is causing the windows builds to fail

```
[6/7][ 85%][8.534s] Install the project...
-- Install configuration: "Release"
-- Installing: T:/Program Files/Swift/Platforms/Windows.platform/Developer/SDKs/WindowsExperimental.sdk/usr/lib/swift_static/windows/x86_64/libswift_Builtin_float.lib
CMake Error at clang/cmake_install.cmake:45 (file):
  file INSTALL cannot find
  "T:/x86_64-unknown-windows-msvc/ExperimentalOverlay/clang/_Builtin_float.swif	module":
  No error.
Call Stack (most recent call first):
  cmake_install.cmake:42 (include)
```

```suggestion
  install(DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/${module_name}.swiftmodule"
```

---

f